packages/identity/android/build.gradle

```diff
@@ -62,7 +62,7 @@ dependencies {
     androidTestImplementation "androidx.test.ext:junit:$androidxJunitVersion"
     androidTestImplementation "androidx.test.espresso:espresso-core:$androidxEspressoCoreVersion"
 
-    implementation 'com.stripe:identity:20.28.2'
+    implementation 'com.stripe:identity:20.28.+'
```

Comment:
Here is Identity's React Native native Android version: https://github.com/stripe/stripe-react-native/blob/d565a0b1562b16ffcbf0932ef1ec4cbfb6b1e18f/android/build.gradle#L139

and its actual value: https://github.com/stripe/stripe-react-native/blob/d565a0b1562b16ffcbf0932ef1ec4cbfb6b1e18f/android/gradle.properties#L3